### PR TITLE
[#1] Add build structure

### LINKED ISSUE #1: Add build structure

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

closes #1

### CODE REVIEW

File: extensions/blkstorage/store.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*
```
Comment (bstasyszyn) [inline extensions/blkstorage/store.go]: Should code be mixed with build scripts in one commit?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	@scripts/dependencies.sh
+
+checks: version depend lint
+
+lint:
+	@scripts/check_lint.sh
+
+all: checks fabric-unit-test
+
+fabric-unit-test: export FABRIC_COMMAND=unit-test
```
Comment (troyronda) [inline Makefile]: there are two targets with the same name here?
also - are we expecting another type of unit-test?

**Comment by securekeybuild** [comment]
verify-unitest

**Comment by securekeybuild** [comment]
verify-unitest

File: Makefile (hunk 2 above)
Comment (fqutishat) [reply Makefile]: done

File: extensions/blkstorage/store.go (hunk 1 above)
Comment (fqutishat) [reply extensions/blkstorage/store.go]: done

**Comment by fqutishat** [comment]
verify-unitest

**Comment by fqutishat** [comment]
verify-unitest

File: Makefile (hunk 2 above)
Comment (troyronda) [reply Makefile]: I mean should the command in line 30 be part of the target in line 31?

Comment (fqutishat) [reply Makefile]: I can't set env var within the same target I had to create another one with same name

**Pull request merged**